src/main/java/com/commonwealthrobotics/controls/SelectionSession.java

```diff
@@ -791,26 +791,27 @@ public void set(TitledPane shapeConfiguration, Accordion shapeConfigurationBox,
 
 	}
 
-	public void clearAllignObjectCache() {
-		getControls().clearAllign();
+	public void clearAlignObjectCache() {
+		getControls().clearAlign();
 	}
 
 	private void setupSnapGrid() {
+        double inch = 25.4; // 1 inch is 25.4mm
 		List<String> grids = Arrays.asList(
-				String.format(Locale.US,"%.4f",0.1)+" mm",
-				String.format(Locale.US,"%.4f",0.25)+" mm", 
-				String.format(Locale.US,"%.4f",0.5)+" mm",
-				String.format(Locale.US,"%.4f",1.0)+" mm", 
-				String.format(Locale.US,"%.4f",(25.4 / 16.0))+" mm (1/16th inch)", 
-				String.format(Locale.US,"%.4f",2.0)+" mm", 
-				String.format(Locale.US,"%.4f",(25.4 / 8.0))+" mm (1/8th inch)", 
-				String.format(Locale.US,"%.4f",5.0)+" mm", 
-				String.format(Locale.US,"%.4f",(25.4 / 4.0))+" mm (1/4 inch)",
-				String.format(Locale.US,"%.4f",8.0)+" mm (Brick)", 
-				String.format(Locale.US,"%.4f",10.0)+" mm",
-				String.format(Locale.US,"%.4f",(25.4 / 2.0))+" mm (1/2 inch)",
-				String.format(Locale.US,"%.4f",20.0)+" mm",
-				String.format(Locale.US,"%.4f",(25.4 ))+" mm (1 inch)"
+				String.format(Locale.US, "%.1f mm", 0.1000),
```

Comment:
I preferred to have the same number of  zeros to make the menu more easily readable.  The trailing zeros spaced out all of the items so that the "inch" measurements are all lined up. I like to specify that metric is "real" measurements, and inches are just an irregular fraction of milimeters.